src/components/file/file.tsx

```diff
@@ -80,6 +81,12 @@ export class File {
     @Prop({ reflect: true })
     public disabled: boolean = false;
 
+    /**
+     * The [accepted file types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#unique_file_type_specifiers)
+     */
+    @Prop({ reflect: true })
+    public accept: string = '*';
```

Comment:
(This isn't an issue I think should prevent merging this, just a small docs thing Johan and I might want to look into.)

Hmm… despite a default value being set here, it doesn't show up in the docs:

![image](https://user-images.githubusercontent.com/154725/110294085-acec2300-7fef-11eb-91e7-1fa460b4cb6d.png)

@jgroth Do you know off the top of your head what the reason might be? I've looked through some of our other components, but we don't have many string properties with default values. We do have `type` on limel-slider, which defaults to `''`, and that shows up as "Default value: `false`" in the docs… so I think the value is being handled incorrectly somewhere. I'm not sure if it's in Kompendium or in some lib we're using though…

![image](https://user-images.githubusercontent.com/154725/110295169-17ea2980-7ff1-11eb-8f3a-2a177a4235c4.png)

![image](https://user-images.githubusercontent.com/154725/110295209-246e8200-7ff1-11eb-8554-5ad52ee5a15f.png)